test/topology_custom/suite.yaml

```diff
@@ -1,4 +1,6 @@
 type: Topology
+# Relative load of this suite, higher values will be scheduled first
```

Comment:
There is test.py manual in docs/dev. It needs to be extended with this new concept. Besides, what happens if there is no load in suite.yaml? What's the defualt?